File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.c
#include <stdio.h>

int main() {
	int N, a, sum, result, count=0;
	scanf("%d", &N);
	if (N >= 0 && N <= 99) {
		if (N < 10) {
				N *= 10;
		}
		a = N;
		do {
			sum = a / 10 + a % 10;
			result = (a % 10) * 10 + sum % 10;
			count += 1;
			a = result;
		} while (result != N);
	}
	printf("%d", count);
	return 0;
}

File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.py
N=int(input())
count=0
if(N>=0 and N<=99):
    if(N<10):
        N*=10
    a=N
    while True:
        add_val=a//10+a%10
        result=(a%10)*10+add_val%10
        a = result
        count+=1;
        if(result== N):
            break
print(count)